Doc/library/sqlite3.rst

```diff
@@ -2440,15 +2436,16 @@ Here's an example of a factory that return a :class:`~collections.namedtuple`.
    >>> row.b   # Attribute access.
    2
 
-As an optimisation, decorate :func:`!_make_cls` with
-:func:`!functools.lru_cache` to avoid creating multiple named tuples
-with identical column spec.
-See :func:`functools.lru_cache` for more details.
-
 With some adjustments, the above recipe can be adapted to use a
 :class:`~dataclasses.dataclass`, or any other custom class,
 instead of a :class:`~collections.namedtuple`.
 
+As an exercise left for the reader, the above example can be optimised
```

Comment:
Since this is a how-to, with the goal of telling a reader how to complete a task, as opposed to a tutorial with the primary aim of helping the reader learn, I'm not sure the "exercise for the reader" framing, which at least to me implies the latter and is generally associated with an academic/workshop/learning context, is a good fit here. Furthermore, IMO its not really necessary, and is more concise and direct without it. 

```suggestion
The above example can be optimised
```

Reply:
How about turning it into a footnote? Or dropping it entirely?

Reply:
You mean this whole paragraph? Yeah, I'm fine with either, since to be honest its content seems mostly orthogonal to anything directly related to the module.

Its also unclear how practically impactful the optimization is in most situations; on Python 3.10 on a old laptop, running `collections.namedtuple('Row', [f'field{idx}' for idx in range(n_cols]` with `timeit`, I get a time of 199 us per execution for 10 cols, or 1.23 ms for 100 cols. This certainly isn't completely _trivial_, if its call in a tight hot loop, but seems unlikely to be a constraint in most real-world situations, given by contrast, even with a trivial in memory example and using "toy" constant SQL statements with no actual db access, running `con.execute("SELECT " + ', '.join([f'{idx} AS column{idx}' for idx in range(100)])).fetchone()` simulating an equivalent db with 100 rows takes 140 us per execution with the default tuple.

Reply:
Oh, I'm pretty you'll find it has a performance impact ;) I'm totally fine with dropping it.